capstone/src/main/webapp/js/autocomplete.js

```diff
@@ -88,6 +88,32 @@ function closeAllElements() {
   }
 }
 
+/**
+ * Keeps track of chosen colleges
+ * @param {Array<string, collegeId>} validSuggestion
+ */
+function keepTrackOfChosenColleges(validSuggestion) {
+  const parent = document.getElementById('suggestions');
+  const listElement = document.createElement('li');
+  listElement.setAttribute('id', validSuggestion.collegeId.id);
+  listElement.innerHTML = validSuggestion.collegeName;
+  listElement.innerHTML += ' ';
+  const removeButton =
+    '<button onclick=\'removeCollege('+
+    validSuggestion.collegeId.id + ')\'>X</button';
+  listElement.innerHTML += removeButton;
```

Comment:
Consider using the DOM to create the button:
```javascript
const removeButton = document.createElement('button');
removeButton.onClick = () => removeCollege(validSuggestion.collegeId.id);
removeButton.innerText = 'X' // or the replacement you choose
listElement.append(removeButton);
```

Reply:
updated :)